cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/lineage/Lineage.java

```diff
@@ -27,27 +26,15 @@
  */
 public class Lineage {
   private final Set<Relation> relations;
-  private final Set<Id.Program> programs;
```

Comment:
how come these two were removed? Just the fact that they can be derived from the set of relations? I think they provided a nice utility, instead of users having to compute this themselves?